lib/transformJiraIssues.js

```diff
@@ -0,0 +1,82 @@
+import { readFileSync, writeFileSync } from "fs";
+
+const jiraIssues = JSON.parse(readFileSync("./jiraIssues.json", "utf-8"));
+
+console.log(jiraIssues.length, "issues");
+
+const ASSIGNEE = {
+  "chrispre@microsoft.com": "chrisspre",
+  "George.Ericson@Dell.com": "GEricson",
+  "Heiko.Theissen@sap.com": "HeikoTheissen",
+  "hubert.heijkers@nl.ibm.com": "Hubert-Heijkers",
+  "jinling@ca.ibm.com": "Hubert-Heijkers", //TODO: add to odata-specs repository
+  "martin.zurmuehl@sap.com": "zurmuehl",
+  "mikep@microsoft.com": "mikepizzo",
+  "ralf.handl@sap.com": "ralfhandl",
+  "stefan@drees.name ": "sthagen",
+};
+
+const gitHubIssues = [];
+
+for (const j of jiraIssues) {
+  const g = {
+    title: j.fields.summary,
```

Comment:
```suggestion
    title: `${j.key}: ${j.fields.summary}`,
```
Have the old Jira key in the title

Reply:
I did not want the old/migrated issues to stick out, so I only attached a backlink to the end of the description/body.

Reply:
We could add a label "from Jira" to the migrated issues so we can easily find them via issue filters, and remove the ODATA-xxx prefix.